ingestion/src/metadata/ingestion/source/pipeline/airflow/metadata.py

```diff
@@ -264,11 +268,30 @@ def get_pipelines_list(self) -> Iterable[OMSerializedDagDetails]:
             json_data_column,
             SerializedDagModel.fileloc,
         ).all():
-            yield OMSerializedDagDetails(
-                dag_id=serialized_dag[0],
-                data=serialized_dag[1],
-                fileloc=serialized_dag[2],
-            )
+            try:
+                dag = AirflowDagDetails(
+                    dag_id=serialized_dag[0],
+                    fileloc=serialized_dag[2],
+                    data=AirflowDag(**serialized_dag[1]),
+                    max_active_runs=serialized_dag[1]["dag"].get(
```

Comment:
I see that we are using quite a lot `serialized_dag[1]["dag"]`. It might be worth it to have a separated variable holding this value and properly validating that it has the right shape (the key exists, etc)

Reply:
Ok 👍